sending-jwt-upstream/README.md

```diff
@@ -33,12 +33,12 @@ definitions {
 
 **Note:** For simplicity we use HS256 here. For production setups we recommend RSA based signatures using a private key for JWT signing and a public key for signature validation.
 
-Now we go to https://jwt.io/ and fill our secret key into the field labeled "VERIFY SIGNATURE" in the right ("Decoded") column.
+Now we go to `https://jwt.io/` and fill our secret key into the field labeled "VERIFY SIGNATURE" in the right ("Decoded") column.
 
 Then we use a token created by the service. We copy the JWT from the box in the left ("Encoded") column and send it in the `Authorization` header:
 
 ```sh
-curl -i -H "Authorization: Bearer ey…" "localhost:8080/private/headers"
+$ curl -i -H "Authorization: Bearer ey…" "localhost:8080/private/headers"
```

Comment:
I prefer removing the `$` since it is just a decorator here. I would like to use the copy Button for code blocks. This gets copied too which then I have to remove anyways. Also the curl output should be separated from the command due to copy purposes.